File: dayTwo/JavaScript/levelTwo/TaskFour.js
const prompt = require("prompt-sync")();

let firstNumber = Number(prompt("Enter first number: "));
let secondNumber = Number(prompt("Enter second number: "));
      
if(firstNumber > secondNumber)
    console.log(firstNumber);  
else if(firstNumber < secondNumber)
    console.log(secondNumber);              

      
    

File: dayTwo/Java/levelTwo/TaskFour.java
import java.util.Scanner;
public class TaskFour{

    public static void main(String[] args){
    
        Scanner inputCollector = new Scanner(System.in);
        System.out.print("Enter first number: ");
        int firstNumber = inputCollector.nextInt();
        System.out.print("Enter second number: ");
        int secondNumber = inputCollector.nextInt();
      
        if(firstNumber > secondNumber)
            System.out.print(firstNumber);  
        else if(firstNumber < secondNumber)
            System.out.print(secondNumber);  
    
    }

}
